readme.md

```diff
@@ -36,10 +48,10 @@
 
 ## Contributing
 
-Modify `source.fish` using Rosé Pine variables, then build variants:
+Modify `_rose_pine_source.fish` using Rosé Pine variables, then build variants:
```

Comment:
I'm actually not sure if this rename is necessary. Reading https://github.com/jorgebucaran/fisher/issues/651#issuecomment-796904618 gives me the impression that a top level source.fish would simply be ignored.

_If you tested this and I'm wrong, please ignore :)_

Reply:
I actually haven't tested it, I just renamed it to be "future" proof. I though that a `source.fish` looked like a reserved filename for fish. Let's move it to `source.fish` 🙌